fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1195,6 +1195,9 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
     @VariableMgr.VarAttr(name = REWRITE_OR_TO_IN_PREDICATE_THRESHOLD, fuzzy = true)
     private int rewriteOrToInPredicateThreshold = 2;
 
+    @VariableMgr.VarAttr(name = "push_topn_to_agg", fuzzy = false, needForward = true)
+    public boolean pushTopnToAgg = true;
```

Comment:
it will be bad case when the input stream data is ordered by desc and the generated filter can't filter any data any all and introduce the additional cost compared with original execution. 
so can we do a data measurement for the worst case and let's know when will happen?

Reply:
we do not have any information about the real order